site/DataPageV2Content.tsx

```diff
@@ -137,6 +142,17 @@ export const DataPageV2Content = ({
         ? `“Data Page: ${datapageData.title}”, part of the following publication: ${datapageData.primaryTopic.citation}. Data adapted from ${producers}. Retrieved from ${canonicalUrl} [online resource]`
         : `“Data Page: ${datapageData.title}”. Our World in Data (${currentYear}). Data adapted from ${producers}. Retrieved from ${canonicalUrl} [online resource]`
 
+    const getImageUrl = (research: DataPageRelatedResearch) => {
```

Comment:
Just a nit, but I think this would be a bit nicer if every one of the three code paths just directly returned its url, and we wouldn't need the `url` var?